app/controllers/course/assessment/submission_question/submission_questions_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class Course::Assessment::SubmissionQuestion::SubmissionQuestionsController < \
+  Course::Assessment::SubmissionQuestion::Controller
+  # Gets the 10 most recent past answers of a specific question
+  def past_answers
+    answers_to_load = past_answers_params[:answers_to_load] || 10
```

Comment:
Left it here in the controller instead of using default arguments. `past_answers_params[:answers_to_load]` gets passed through as `nil` actually, so we'll need to check if `nil` to use `@submission_question.past_answers` vs `@submission_question.past_answers(answers_to_load)`. This way is clearer.